auto_tune_vllm/utils/vllm_cli_parser.py

```
@@ -0,0 +1,515 @@
+        cleaned_desc = re.sub(default_pattern, '', description).strip()
+        
+        # Parse different default value types
+        if default_str.lower() in ['none', 'null']:
+            return None, cleaned_desc
+        elif default_str.lower() in ['true', 'false']:
+            return default_str.lower() == 'true', cleaned_desc
+        elif default_str.startswith('[') and default_str.endswith(']'):
+            # List default
+            try:
+                return ast.literal_eval(default_str), cleaned_desc
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)